--- bcm43xx-20060120/drivers/net/wireless/bcm43xx/bcm43xx.h~	2006-01-20 06:00:12.000000000 +0100
+++ bcm43xx-20060120/drivers/net/wireless/bcm43xx/bcm43xx.h	2006-01-20 17:52:51.000000000 +0100
@@ -7,8 +7,8 @@
 #include <linux/interrupt.h>
 #include <linux/stringify.h>
 #include <linux/pci.h>
-#include <net/ieee80211.h>
-#include <net/ieee80211softmac.h>
+#include "net/ieee80211.h"
+#include "net/ieee80211softmac.h"
 #include <asm/atomic.h>
 #include <asm/io.h>
 
